packages/cli/src/commands/doctor/types.ts

```diff
@@ -32,12 +32,14 @@ export type EnvironmentInfo = {
     'iOS SDK': {
       Platforms: string[];
     };
-    'Android SDK': {
-      'API Levels': string[];
-      'Build Tools': string[];
-      'System Images': string[];
-      'Android NDK': string;
-    };
+    'Android SDK':
```

Comment:
Ideally, we should contribute these typings to the `envinfo` repo. There are no typings yet in there. There is [an ongoing PR](https://github.com/tabrindle/envinfo/pull/105), but the typings are quite incomplete.